payload/Library/Scripts/Install or Defer.sh

```diff
@@ -309,8 +313,26 @@ install_updates () {
 
         done
 
-    else
+    elif [ "$SYSPREFS_UPDATE" -eq 1 ] && (( DEFER_TIME_LEFT > 0 )); then
```

Comment:
This section looks identical to the Apple Silicon workflow at this point minus the loop, but I think the intention is to persistently display the update alert which will require some kind of loop. Could probably just add this as another option for [the Apple Silicon check](https://github.com/mpanighetti/install-or-defer/pull/80/files#diff-18dd8ed5be044b8a12133aa395df69912ddb9ff6df44c8280f7b903633f45f1bR290), e.g. `if [ "$PLATFORM_ARCH" = "arm64" ] || [ "$SYSPREFS_UPDATE" -eq 1 ]; then` (and change the `echo` line underneath to indicate the other possible cause of the persistent alert).

Reply:
Yep, that seems wise, and reduces the amount of extraneous code. Originally I figured the messaging didnt need to be quite as aggressive/persistent as the M1s (because unlike M1s, the script CAN try to automatically install the updates at the end of the deferral window), but I agree that keeping it consistent makes sense. I've updated the PR.